server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -309,7 +310,7 @@ public TSFetchMetadataResp fetchMetadata(TSFetchMetadataReq req) {
           status = new TSStatus(getStatus(TSStatusCode.SUCCESS_STATUS));
           break;
         case "SHOW_STORAGE_GROUP":
-          Set<String> storageGroups = getAllStorageGroups();
+          Set<String> storageGroups = new HashSet<>(getAllStorageGroups());
```

Comment:
I think it's unnecessary to wrap `getAllStorageGroups()` with a `new HashSet<>`

Reply:
The data structure required by the upper interface is a `Set`, while the lower interface returns a `List`, so I have to do the conversion.
But actually, it does not need to be a set, as we won't have duplicated SGs. I will change the upper interface to receive a `List` later.